src/api/libcellml/specificationrules.h

```diff
@@ -27,9 +27,17 @@ namespace libcellml {
 enum class SpecificationRule
 {
     UNDEFINED,
+    DATA_REPR_IDENTIFIER_UNICODE,
+    DATA_REPR_IDENTIFIER_LATIN_ALPHANUM,
+    DATA_REPR_IDENTIFIER_ATLEAST1_ALPHANUM,
+    DATA_REPR_IDENTIFIER_BEGIN_EURO_NUM,
+    DATA_REPR_IDENTIFIER_IDENTICAL,
+    DATA_REPR_NNEG_INT_BASE10,
+    DATA_REPR_NNEG_INT_EURO_NUM,
     MODEL_ELEMENT,
     MODEL_NAME,
     MODEL_CHILD,
+    MODEL_MORE_THAN_1_ENCAPSULATION,
```

Comment:
Replace with `MODEL_MORE_THAN_ONE_ENCAPSULATION` and the same elsewhere where `1` is used.